sysutils/libvirt/Portfile

```diff
@@ -50,97 +48,77 @@ depends_lib         port:curl \
 # error: You need at least XCode Clang v5.1 to compile QEMU
 compiler.blacklist-append apple-gcc* gcc-3.3 *gcc-4.* macports-clang-3.3 {clang < 503}
 
+patchfiles-append   patch-meson.diff
+
 # libvirt build scripts require python 3.x
 configure.python    ${prefix}/bin/python3.8
 configure.perl      ${prefix}/bin/perl5
-
-configure.args      ac_cv_path_RST2HTML=${prefix}/bin/rst2html-3.8.py \
-                    ac_cv_path_RST2MAN=${prefix}/bin/rst2man-3.8.py \
-                    --disable-silent-rules \
-                    --enable-debug \
-                    --without-apparmor \
-                    --without-attr \
-                    --without-audit \
-                    --with-bash-completion \
-                    --with-bash-completions-dir=${prefix}/share/bash-completion/completions \
-                    --without-bhyve \
-                    --without-blkid \
-                    --without-capng \
-                    --with-curl \
-                    --without-dbus \
-                    --with-driver-modules \
-                    --without-dtrace \
-                    --with-esx \
-                    --without-fuse \
-                    --without-glusterfs \
-                    --without-hal \
-                    --without-hyperv \
-                    --with-init-script=none \
-                    --without-libpcap \
-                    --without-libssh \
-                    --with-libvirtd \
-                    --without-libxl \
-                    --without-lxc \
-                    --without-macvtap \
-                    --without-netcf \
-                    --without-network \
-                    --without-numactl \
-                    --without-openvz \
-                    --without-openwsman \
-                    --without-pm-utils \
-                    --without-polkit \
-                    --without-qemu \
-                    --with-readline \
-                    --with-remote \
-                    --without-sanlock \
-                    --without-sasl \
-                    --with-secrets \
-                    --without-selinux \
-                    --without-ssh2 \
-                    --without-sysctl \
-                    --with-test \
-                    --without-udev \
-                    --with-vbox \
-                    --without-virtualport \
-                    --with-vmware \
-                    --without-vz \
-                    --without-wireshark-dissector \
-                    --with-yajl
-
-# As of 5.10.0, an out-of-source build is required.
-configure.cmd       ../${worksrcdir}/autogen.sh --no-git
-configure.dir       ${workpath}/build
-build.dir           ${configure.dir}
-post-extract {
-    file mkdir ${configure.dir}
-}
+configure.env       PATH=${prefix}/bin:$env(PATH) \
+                    PKG_CONFIG_PATH=${prefix}/lib/pkgconfig
```

Comment:
${prefix}/bin is assumed to be in PATH already so this line isn't needed. ${prefix}/lib/pkgconfig is already in the default pkg-config search path so you don't need to specify it. If you did need to specify it, you should use `configure.pkg_config_path` rather than putting PKG_CONFIG_PATH into the environment manually.

Reply:
Thanks for the info, indeed it works without these two lines. For some reason the first time it did not work for me. I'll remove it.